first-year-parse/textExtract.py

```diff
@@ -0,0 +1,14 @@
+import textract
+# import regex as re
+
+text = str(textract.process('/home/sriyash/chillZone/Aut_Time_Table_19_20.docx'))
```

Comment:
Is there any reason you decided not to use `PyPDF2`?

Reply:
It's just that I didn't know about it. And non-uniformity in layout like no of columns rows across the different tables makes preprocessing necessary so I could just use PyPDF2 to extract the text.

Reply:
Ah, I see.

I think it's better to use PyPDF2 compared to Textract because it reduces one pre-processing step (converting from PyPDF2 to Docx) On computers with Word, most people would have to just `Ctrl-A`, Copy, Paste into LibreOffice and save as Docx. Being able to pass in PDF directly would be less cumbersome :slightly_smiling_face:

Reply:
ok. Implementing the changes

Reply:
Cannot use PyPDF2 because it takes the text to be a part of the image so extractText() on any page yields a NULL string. I probably used an online convertor with OCR.

Reply:
Please put the exact steps for that as well so the person can do. It is always better to put the exact steps and the tools that you have used.

Reply:
@sriyash421 What about this?

Reply:
I have put this in the readme

Reply:
I have mentioned the link to the service to generate the .docx file in the README